content/en/blog/_posts/non-root-containers-and-devices.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: blog
+title: 'Non-root Containers And Devices'
+date: 2021-09-YY
+slug: non-root-containers-and-devices
+---
+
+**Author:** Mikko Ylinen (Intel)
+
+## Abstract
```

Comment:
Headings need to be more readable and organic. This section could instead be titled "How Kubernetes supports the principle of least privilege" (for example)